wordpress-mobile/WordPress-Aztec-iOS
# Improves the undo manager code.

Relates to #60 

Improves the undo management code to avoid having to propagate `UndoManager` to every single node in the DOM tree.

Unit tests have to provide an empty block, since nodes are not supposed to exist without an undo block (by design, at least in this v1 of undo management - we can review this later on).

**How to test:**

- Delete text from single node.
- Move the caret around and repeat.
- Shake to undo any times you deem necessary.
- Check the HTML is as expected.
- Shake to redo any times you deem necessary.
- Check the HTML is as expected.
